scripts/get-repos/index.js

```diff
@@ -0,0 +1,34 @@
+import {Octokit} from '@octokit/rest'
+
+const LANGUAGES = ["JavaScript", "Go"]
+const ORGS = ["ipfs", "ipfs-shipyard", "ipld", "libp2p", "multiformats"]
+
+const options = {}
+if (process.env.GITHUB_TOKEN) {
+  options.auth = process.env.GITHUB_TOKEN
+}
+
+const octokit = new Octokit(options)
+
+async function list_repos(orgs, langs) {
+  let results = []
+  for (const org of orgs) {
+    for await (const resp of octokit.paginate.iterator(octokit.rest.repos.listForOrg, {org})) {
+      for (const repo of resp.data) {
+        if (repo.private || repo.archived) {
+          continue
+        }
+        if (!langs.includes(repo.language)) {
+          continue
+        }
+        results.push(repo.full_name)
```

Comment:
Note that I slightly changed the output here to make this script a little more universally applicable. It now just outputs a list of repos (strings), not wrapped in a `{ target: "<repo>" }` object.